src/set_theory/game/nim.lean

```diff
@@ -303,73 +297,46 @@ lemma grundy_value_eq_mex_right : ∀ (G : pgame) [G.impartial],
   apply grundy_value_neg
 end
 
+/-- The Grundy value of the sum of two nim games with natural numbers of piles equals their bitwise
+xor. -/
+-- Todo: this actually generalizes to all ordinals, by defining `ordinal.lxor` as the pairwise
+-- `nat.lxor` of base `ω` Cantor normal forms.
```

Comment:
I guess we can also say it's the bitwise `xor` in base 2 Cantor normal form, because we have `ω^o = 2^(ω*o)` ...